.github/workflows/build.yml

```diff
@@ -302,4 +303,4 @@ jobs:
           npx semantic-release --dry-run ${{github.ref != 'refs/heads/master'}} --ci ${{github.ref == 'refs/heads/master'}}
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@ViralRuparel looks like a bad merge here, you should probably revert your changes to this file

Reply:
yes something went wrong while merging, let me fix!